network_diffusion/mln/functions.py

```diff
@@ -524,3 +526,21 @@ def get_toy_network_cim() -> MultilayerNetwork:
     return MultilayerNetwork.from_nx_layers(
         [layer_1, layer_2, layer_3], ["l1", "l2", "l3"]
     )
+
+
+def draw_mln(net: MultilayerNetwork, dpi: int | None = None) -> None:
+    """Draw briefly a given multilayer network."""
+    if net.get_actors_num() > 100:
+        warnings.warn(
+            f"Too large network ({net.get_actors_num()}). \
+                Visualisation can be crippled.",
+            stacklevel=1,
+        )
+    pos = nx.drawing.shell_layout([a.actor_id for a in net.get_actors()])
+    fig, axs = plt.subplots(nrows=1, ncols=len(net.layers))
+    fig.set_dpi(dpi if dpi else 300)
```

Comment:
(1) dpi: int = 300; fig.set_dpi(dpi). One if less

Reply:
Yeah, too much baroque in the code doesn't make it more efficient ;)